types/jest-image-snapshot/jest-image-snapshot-tests.ts

```diff
@@ -1,10 +1,13 @@
+import { expect } from 'expect';
 import {
     configureToMatchImageSnapshot,
     MatchImageSnapshotOptions,
     toMatchImageSnapshot,
     updateSnapshotState,
 } from 'jest-image-snapshot';
 
+declare const it: any;
```

Comment:
Could this be typed through `@types/jest`? Or perhaps simply to `import {it} from '@jest/globals'`?

Reply:
The value of `it` is actually irrelevant. It just makes the tests look more representative. If it’s imported from `@types/jest` or `@jest/globals`, this would add a dependency (dev dependencies are not allowed in DefinitelyTyped).

Reply:
Fine with me. It just felt like a regression. I am not a maintainer of any `@types/*` packages, just wanted to draw attention.